build.gradle

```diff
@@ -20,9 +20,9 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.0.0'
     automatticTracksVersion = '2.2.0'
-    gutenbergMobileVersion = 'v1.92.0'
+    gutenbergMobileVersion = 'v1.93.0-alpha1'
     wordPressAztecVersion = 'v1.6.3'
-    wordPressFluxCVersion = '2.23.1'
+    wordPressFluxCVersion = 'trunk-8b2fc7fdc3fe4e864a065210b356c20f8882f043'
```

Comment:
In order to include the change https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/4c38c0da2445f4136a92a587414e9e4a27e5e534 relate to this PR and the hotfix `2.23.1`, I updated the reference to https://github.com/wordpress-mobile/WordPress-FluxC-Android/commit/8b2fc7fdc3fe4e864a065210b356c20f8882f043 to include both.

cc @thomashorta as you created the hotfix in case you have any thoughts/concerns. Thanks!